packages/contracts-bedrock/test/properties/PROPERTIES.md

```diff
@@ -0,0 +1,72 @@
+# supertoken properties
```

Comment:
you include invariants from liquidity migration. Wdyt about renaming this title?

Reply:
*and factory